test/system/search_test.rb

```diff
@@ -10,4 +10,13 @@ class SearchTest < ApplicationSystemTestCase
 
     assert_selector('h2', text: 'Results for Canon')
   end
+  
+  test 'searching using navbar autocomplete' do
+    visit '/'
+
+    fill_in("searchform_input", with: "Canon")
+
+    assert page.evaluate_script("$('.typeahead.dropdown-menu').is(':visible')")
```

Comment:
Looks like Capybara won't wait unless we ask it to look for a selector. Even then we can increase the max time:

`select('Item that is being loaded', wait: 10)`

Reply:
https://stackoverflow.com/questions/36536111/waiting-for-ajax-with-capybara-poltergeist

Reply:
also see https://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Matchers#assert_selector-instance_method

for example: `page.assert_selector('li', text: 'Horse', visible: true)`